client/lib/gsuite/new-users.ts

```diff
@@ -86,10 +85,38 @@ const validateOverallEmailAgainstExistingEmails = (
 			: mailBoxError,
 } );
 
+const clearPreviousErrors = ( users: GSuiteNewUser[] ) => {
+	return users.map( user => mapValues( user, field => removePreviousErrors( field ) ) );
+};
+
+const validateNewUserMailboxIsUnique = (
+	{ value: mailBox, error: previousError }: GSuiteNewUserField,
```

Comment:
`mailBox` should probably be written as `mailbox` to be consistent with `mailboxesByCount`.